tests/HtmlElementTest.php

```diff
@@ -85,4 +85,35 @@ function it_generates_elements_with_boolean_attributes()
 
         $this->assertSame('<input required>', $element->render());
     }
+
+    /** @test */
+    function it_generates_a_void_element_with_mixed_attributes()
+    {
+        $element = new HtmlElement(
+            'input',
+            ['id' => 'my_paragraph', 'class' => 'paragraph', 'required'],
```

Comment:
Although probably not very important, I'd change the name of the id and the class.